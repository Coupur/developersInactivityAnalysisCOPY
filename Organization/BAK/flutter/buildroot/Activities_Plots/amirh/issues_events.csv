id,date,event,creator_login
2032290303,2018-12-18 08:20:22,mentioned,amirh
2032290304,2018-12-18 08:20:22,subscribed,amirh
1344232077,2017-11-16 01:37:53,review_requested,amirh
1344238311,2017-11-16 01:44:30,head_ref_force_pushed,amirh
1344239139,2017-11-16 01:45:23,referenced,amirh
1344239141,2017-11-16 01:45:23,merged,amirh
1344239145,2017-11-16 01:45:23,closed,amirh
1341794523,2017-11-14 21:20:12,review_requested,amirh
1343419656,2017-11-15 16:50:43,head_ref_force_pushed,amirh
1343423634,2017-11-15 16:52:31,head_ref_force_pushed,amirh
1344057756,2017-11-15 22:56:49,referenced,amirh
1344057757,2017-11-15 22:56:49,merged,amirh
1344057763,2017-11-15 22:56:49,closed,amirh
