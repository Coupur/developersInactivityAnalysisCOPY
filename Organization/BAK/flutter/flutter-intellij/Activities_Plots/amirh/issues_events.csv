id,date,event,creator_login
1803735548,2018-08-22 21:13:51,mentioned,amirh
1803735549,2018-08-22 21:13:51,subscribed,amirh
