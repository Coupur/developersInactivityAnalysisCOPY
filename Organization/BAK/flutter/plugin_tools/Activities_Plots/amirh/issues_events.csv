id,date,event,creator_login
2452234944,2019-07-01 19:35:21,mentioned,amirh
2452234945,2019-07-01 19:35:21,subscribed,amirh
