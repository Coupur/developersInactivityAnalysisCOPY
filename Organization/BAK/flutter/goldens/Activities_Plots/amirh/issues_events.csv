id,date,event,creator_login
1768906594,2018-08-03 00:05:17,referenced,amirh
1768906595,2018-08-03 00:05:17,merged,amirh
1768906601,2018-08-03 00:05:17,closed,amirh
1768906640,2018-08-03 00:05:20,head_ref_deleted,amirh
